includes/PoolCounter.php

```diff
@@ -161,7 +161,9 @@ function error( $status ) {
 	 * @param $status Status
 	 */
 	function logError( $status ) {
-		wfDebugLog( 'poolcounter', $status->getWikiText() );
+		$msg = json_encode( $status->getErrorsArray() );
+
+		wfDebugLog( 'poolcounter', $msg);
```

Comment:
Or even better, normalize `Status` objects in `WikiaLogger`. We do that for exceptions - https://github.com/Wikia/app/blob/dev/lib/composer/monolog/monolog/src/Monolog/Formatter/NormalizerFormatter.php#L81-L83